wondrous-app/components/Header/index.tsx

```diff
@@ -1,15 +1,19 @@
 import { useRouter } from 'next/router';
-import { useMutation } from '@apollo/client';
 
-import { MARK_ALL_NOTIFICATIONS_READ, MARK_NOTIFICATIONS_READ } from 'graphql/mutations/notification';
 import { useIsMobile, useGlobalContext } from 'utils/hooks';
-import { useMe, withAuth } from '../Auth/withAuth';
-import HeaderMemo from './HeaderMemo';
+import { useMe, withAuth } from 'components/Auth/withAuth';
+import HeaderMemo from 'components/Header/HeaderMemo';
+import { useContext, useEffect } from 'react';
+import { SnackbarAlertContext } from 'components/Common/SnackbarAlert';
+import { ConnectDiscordLink } from './styles';
+
+const DISCORD_SNACKBAR_DURATION = 1000 * 60 * 10;
```

Comment:
add comment to how long this is

Reply:
isnt this 10 minutes? that's a bit too long, even just 30 seconds is good enough imo